src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +21,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(string -> Arrays.stream(string.split(DIVIDER)))
+                .mapToInt(Integer::parseInt)
+                .filter(StreamPractice::isEven)
```

Comment:
same with every other private method
```suggestion
                .filter(this::isEven)
```